tests/unit/orders-update.spec.js

```diff
@@ -40,3 +33,172 @@ test(`OrdersUpdate
   )
   t.end()
 })
+
+test(`summaryReport
+  should contain no errors and no imports if nothing is imported`, (t) => {
+  const importer = new OrdersUpdate(apiClientConfig)
+  const expected = { errors: [], inserted: [], successfullImports: 0 }
+  const actual = JSON.parse(importer.summaryReport())
+
+  t.deepEqual(actual, expected, 'No errors if no import occurs')
+
+  t.end()
+})
+
+test(`validateOrderData
+  should resolve if the order data is valid`, (t) => {
+  const importer = new OrdersUpdate(apiClientConfig)
+  importer.validateOrderData(orderSample)
+    .then(() => t.end())
+    .catch(t.fail)
+})
+
+test(`validateOrderData
+  should reject if the order data is invalid`, (t) => {
+  const importer = new OrdersUpdate(apiClientConfig)
+  importer.validateOrderData({ id: {} })
+    .then(t.fail)
+    .catch(() => t.end())
```

Comment:
We should check for the proper error message because anything can make it to fail.